composer.json

```diff
@@ -21,29 +21,28 @@
         },
         "sort-packages": true,
         "platform": {
-            "php": "8.1.99"
+            "php": "8.0.99"
         }
     },
     "require": {
-        "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
```

Comment:
@gsteel 
Why?

Reply:
My mistake - Just did a rebase and killed it. Hang on

Reply:
No problem. Good work! 👍🏻